### PR TITLE
Deprecate pyscript output attribute

### PR DESCRIPTION
**Pull request by FabioRosado**
This PR deprecates the `output` attribute in the `py-script` tag and updates the docs that use it.

Closes: #973

### CODE REVIEW

**Comment by JeffersGlass** [comment]
Are these changes also captured in #982 ? From a casual glance it looks like the output deprecation warning and docs are in both.

**Comment by FabioRosado** [comment]
Yes I branched of that other branch since this one would cause a bunch of conflicts haha

File: pyscriptjs/src/components/pyscript.ts [hunk 1]
```diff
@@ -9,6 +9,14 @@ const logger = getLogger('py-script');
 export function make_PyScript(runtime: Runtime) {
     class PyScript extends HTMLElement {
         async connectedCallback() {
+            if (this.hasAttribute('output')) {
+                const deprecationMessage = (
+                    "The 'output' attribute is deprecated. You should use the " +
+                    "'display' API to output the content to a specific element. " +
+                    'For example display(myElement, target="divID").'
```
Comment (antocuni) [inline pyscriptjs/src/components/pyscript.ts:16]:
```suggestion
                    "The 'output' attribute is deprecated and ignored. You should use " +
                    "'display()' to output the content to a specific element. " +
                    'For example display(myElement, target="divID").'
```

Comment (antocuni) [reply pyscriptjs/src/components/pyscript.ts:16]: a bit of nitpick, but I think it's good to underline that it's not only deprecated, it's completely ignored

**Review by antocuni — APPROVED** [review]
LGTM

**Pull request merged**